app-misc/broot/broot-1.6.1.ebuild

```diff
@@ -0,0 +1,234 @@
+getrandom-0.1.16
+getrandom-0.2.3
+gif-0.11.2
+git2-0.13.20
+glassbench-0.3.0
+glob-0.3.0
+hashbrown-0.9.1
+hashlink-0.6.0
+heck-0.3.3
+hermit-abi-0.1.18
+id-arena-2.2.1
+idna-0.2.3
+image-0.23.14
+indexmap-1.6.2
+instant-0.1.9
+is_executable-0.1.2
+itoa-0.4.7
+jobserver-0.1.22
+jpeg-decoder-0.1.22
+lazy-regex-2.2.1
+lazy-regex-proc_macros-2.2.1
+lazy_static-1.4.0
+lazycell-1.3.0
+lfs-core-0.3.4
+libc-0.2.97
+libgit2-sys-0.12.21+1.1.0
+libsqlite3-sys-0.20.1
+libz-sys-1.1.3
+line-wrap-0.1.1
+linked-hash-map-0.5.4
+lock_api-0.4.4
+log-0.4.14
+matches-0.1.8
+memchr-2.4.0
+memmap-0.7.0
+memoffset-0.6.4
+minimad-0.7.1
+minimad-0.8.0
+miniz_oxide-0.3.7
+miniz_oxide-0.4.4
+mio-0.7.13
+miow-0.3.7
+ntapi-0.3.6
+num-integer-0.1.44
+num-iter-0.1.42
+num-rational-0.3.2
+num-traits-0.2.14
+num_cpus-1.13.0
+once_cell-1.8.0
+onig-6.2.0
+onig_sys-69.7.0
+open-1.7.0
+parking_lot-0.11.1
+parking_lot_core-0.8.3
+pathdiff-0.2.0
+percent-encoding-2.1.0
+phf-0.8.0
+phf_generator-0.8.0
+phf_macros-0.8.0
+phf_shared-0.8.0
+pkg-config-0.3.19
+plist-1.1.0
+png-0.16.8
+ppv-lite86-0.2.10
+proc-macro-hack-0.5.19
+proc-macro2-1.0.27
+proc-status-0.1.1
+quote-1.0.9
+rand-0.7.3
+rand-0.8.4
+rand_chacha-0.2.2
+rand_chacha-0.3.1
+rand_core-0.5.1
+rand_core-0.6.3
+rand_hc-0.2.0
+rand_hc-0.3.1
+rand_pcg-0.2.1
+rayon-1.5.1
+rayon-core-1.9.1
+redox_syscall-0.2.9
+redox_users-0.4.0
+regex-1.5.4
+regex-automata-0.1.10
+regex-syntax-0.6.25
+remove_dir_all-0.5.3
+rusqlite-0.24.2
+ryu-1.0.5
+safemem-0.3.3
+same-file-1.0.6
+scoped_threadpool-0.1.9
+scopeguard-1.1.0
+secular-1.0.0
+serde-1.0.126
+serde_derive-1.0.126
+serde_json-1.0.64
+signal-hook-0.1.17
+signal-hook-registry-1.4.0
+siphasher-0.3.5
+smallvec-1.6.1
+splitty-0.1.0
+str-buf-1.0.5
+strict-0.1.4
+strsim-0.8.0
+svg-0.8.2
+syn-1.0.73
+syntect-4.5.0
+tempfile-3.2.0
+termimad-0.10.3
+termimad-0.11.1
+terminal-clipboard-0.2.1
+termux-clipboard-0.1.0
+textwrap-0.11.0
+thiserror-1.0.25
+thiserror-impl-1.0.25
+tiff-0.6.1
+time-0.1.43
+tinyvec-1.2.0
+tinyvec_macros-0.1.0
+toml-0.5.8
+umask-1.0.0
+unicode-bidi-0.3.5
+unicode-normalization-0.1.19
+unicode-segmentation-1.7.1
+unicode-width-0.1.8
+unicode-xid-0.2.2
+url-2.2.2
+users-0.11.0
+vcpkg-0.2.15
+vec_map-0.8.2
+version_check-0.9.3
+walkdir-2.3.2
+wasi-0.9.0+wasi-snapshot-preview1
+wasi-0.10.2+wasi-snapshot-preview1
+weezl-0.1.5
+which-4.1.0
+winapi-0.3.9
+winapi-i686-pc-windows-gnu-0.4.0
+winapi-util-0.1.5
+winapi-x86_64-pc-windows-gnu-0.4.0
+x11-clipboard-0.5.2
+xcb-0.9.0
+xml-rs-0.8.3
+yaml-rust-0.4.5
+"
+
+inherit cargo
+
+DESCRIPTION="A new way to see and navigate directory trees"
+HOMEPAGE="https://dystroy.org/broot/"
+SRC_URI="https://github.com/Canop/broot/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	$(cargo_crate_uris)"
+
+LICENSE="Apache-2.0 BSD-2 BSD LGPL-3+ MIT ZLIB"
+SLOT="0"
+KEYWORDS="~amd64"
+
+QA_FLAGS_IGNORED="usr/bin/broot"
+
+src_prepare() {
+	default
+
+	local mandate=$(date -r man/page +'%Y\/%m\/%d' || die)
+	sed -e "s/#version/${PV}/" \
+		-e "s/#date/${mandate}/" \
```

Comment:
Trivial but rather than escape `\/` in date output, you could change the sed delimiters, aka you can use `s|#date|${mandate}|` and then / can be safely used in it.

Reply:
I tend to use `s,,,` on the command line, because it's a single keypress and not a shell metacharacter, but have heard complaints that it's hard for others to read, so I tend to revert to the default `/` when writing proper code.

But the ebuild is now updated to use `|`.